### ISSUE TITLE
One Player

### ISSUE THREAD

**Issue by johnRedden** [johnRedden/air-hockey-5#1]
Somehow get the one player option to work with the app controlling the second paddle.

I am initially thinking a constant small circle constrained to a paddle of which x, and y values change according to the puck x and y values on update.  

...OR  some lame simple up and down motion in front of the hole.

**Comment by johnRedden** [comment]
Got one player AI going... well VAI.  Visalia AI ... this is the best I can do for now. Totally unbeatable though.  Still need to hobble it a bit and maybe add levels (easy, medium, hard).

**Comment by pgonzbecer** [comment]
Still a hard boss, but I gave him a little more personality and stuff to do. Although We need to dumb it down a tad bit more.

**Comment by pgonzbecer** [comment]
Err Side note: It might be because I'm still using a freakin' trackpad, but it seems 80% of the times I scored, was due to the AI's own moves
